src/modules/Builder/features/ActivitySettings/ScoresAndReports/SectionScoreCommonFields/SectionScoreCommonFields.tsx

```diff
@@ -87,6 +87,7 @@ export const SectionScoreCommonFields = ({
           control={control}
```

Comment:
Could you please add dependencies to useEffect or disable the dependencies check for useEffects?

Reply:
good point! fixed